cmd/rosetta/cli.go

```diff
@@ -166,6 +168,13 @@ VERSION:
 		Usage:    "Specifies the configuration file for custom currencies.",
 		Required: false,
 	}
+
+	cliFlagActivationEpochSpica = cli.UintFlag{
+		Name:     "activation-epoch-spica",
+		Usage:    "Specifies the activation epoch for Spica release.",
+		Required: false,
+		Value:    math.MaxUint32,
```

Comment:
maybe set the default value as the real mainnet activation epoch

Reply:
It will be indeed, but it isn't known for the moment. Good point - I won't forget about this.